node/pkg/watchers/solana/ccq.go

```diff
@@ -37,7 +41,8 @@ func (w *SolanaWatcher) ccqHandleQuery(ctx context.Context, queryRequest *query.
 
 	switch req := queryRequest.Request.Query.(type) {
 	case *query.SolanaAccountQueryRequest:
-		w.ccqHandleSolanaAccountQueryRequest(ctx, queryRequest, req)
+		giveUpTime := start.Add(query.RetryInterval).Add(-250 * time.Millisecond)
```

Comment:
thoughts on `start.Add(query.RetryInterval).Add(-CCQ_FAST_RETRY_INTERVAL)` or something? wondering what the intention with the `-250` is so maybe we could make that more clear :slightly_smiling_face:

Reply:
Created a const and added a comment.